### PR TITLE
no_whitespace_in_blank_line and no_blank_lines_after_phpdoc fixers bug

### PR DESCRIPTION
**Pull request by kubawerlos**
Priorities were wrong and caused problem as showed in test case

### CODE REVIEW

**Comment by keradus** [comment]
super nice, RTM if CIs are green

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Comment by SpacePossum** [comment]
nice catch, thanks @kubawerlos !

File: src/Fixer/Phpdoc/NoBlankLinesAfterPhpdocFixer.php [hunk 1]
```diff
@@ -61,7 +61,8 @@ class Bar {}
     public function getPriority()
     {
         // should be ran before the SingleBlankLineBeforeNamespaceFixer.
-        return 1;
+        // should be ran after the NoWhitespaceInBlankLineFixer.
```
Comment (GrahamCampbell) [inline src/Fixer/Phpdoc/NoBlankLinesAfterPhpdocFixer.php]: should be **run**, not **ran**

**Pull request closed without merging**